specification/azurefleet/AzureFleet.Management/client.tsp

```diff
@@ -2,5 +2,367 @@ import "./main.tsp";
 import "@azure-tools/typespec-client-generator-core";
 
 using Azure.ClientGenerator.Core;
+using Microsoft.AzureFleet;
 
 @@clientName(Microsoft.AzureFleet, "ComputeFleetMgmt", "python");
+
+@@access(Operations.list, Access.internal, "csharp");
+
+@@clientName(ApiError, "ComputeFleetApiError", "csharp");
```

Comment:
Is this a common practice to add Provider Name before types for csharp? Will these now be different from other language sdks? and a cause of concern.

Reply:
.NET SDK has a more strict criteria of model naming than other languages, that's why we are doing renaming for .NET SDK models only here.